public/errors.css

```diff
@@ -0,0 +1,755 @@
+@import url("https://fonts.googleapis.com/css?family=Alfa+Slab+One|Open+Sans:400,400i,700,700i&subset=cyrillic,cyrillic-ext,greek,greek-ext,latin-ext,vietnamese");
```

Comment:
@chrisfrank we usually don't check in compiled CSS. Since the build task covers both inline and main stylesheets, could we remove this from the diff?